jenkins/jobs/cd/tiproxy_docker.groovy

```diff
@@ -0,0 +1,24 @@
+pipelineJob('tiproxy-docker') {
+    definition {
+        cpsScm {
+            lightweight(true)
+            scriptPath('jenkins/pipelines/cd/tiproxy-docker.groovy')
+            scm {
+                git{
+                    remote {
+                        url('git@github.com:PingCAP-QE/ci.git')
+                        credentials('github-sre-bot-ssh')
```

Comment:
Please use https url, and it's public, so no need credentials. Please check rest parts.

Reply:
ref: <img width="622" alt="image" src="https://github.com/PingCAP-QE/ci/assets/2574558/1f659af6-88b2-4465-b837-e84062862dce">